src/Middleware/WebSockets/test/ConformanceTests/Helpers.cs

```diff
@@ -8,21 +8,8 @@ public class Helpers
         public static string GetApplicationPath(string projectName)
         {
             var applicationBasePath = AppContext.BaseDirectory;
-
             var directoryInfo = new DirectoryInfo(applicationBasePath);
-            do
-            {
-                var solutionFileInfo = new FileInfo(Path.Combine(directoryInfo.FullName, "WebSockets.sln"));
-                if (solutionFileInfo.Exists)
-                {
-                    return Path.GetFullPath(Path.Combine(directoryInfo.FullName, "test", projectName));
-                }
-
-                directoryInfo = directoryInfo.Parent;
-            }
-            while (directoryInfo.Parent != null);
-
-            throw new Exception($"Solution root could not be found using {applicationBasePath}");
+            return Path.GetFullPath(Path.Combine(directoryInfo.FullName, "test", projectName));
```

Comment:
```suggestion
            return Path.GetFullPath(Path.Combine(AppContext.BaseDirectory, projectName));
```
I don't think "test" is going to appear in the path anymore, and you can get rid of the directoryInfo variable.